bot/cogs/snakes.py

```diff
@@ -1,21 +1,35 @@
 # coding=utf-8
 import logging
-from typing import Any, Dict
+import re
 
-from discord.ext.commands import AutoShardedBot, Context, command
+import aiohttp
+import bs4
+import discord
+import html2text
+from discord.ext import commands
 
 log = logging.getLogger(__name__)
+API = 'http://en.wikipedia.org/w/api.php?format=json&redirects=1&action='
+
+rSENTENCE = re.compile(r'^.+?\.')
+rBRACK = re.compile(r'[[(].+?[\])]')
 
 
 class Snakes:
     """
     Snake-related commands
     """
 
-    def __init__(self, bot: AutoShardedBot):
```

Comment:
Why did you remove the type hinting?

Reply:
Sorry. My partner was for a short while only able to use python 3.5 (and `typing` was added in 3.6) so I started removing hints preemptively for compatibility -- should've put them back once he was able to get on 3.6, though.